README.md

```diff
@@ -66,6 +66,55 @@ pip install -r requirements.txt
 
 This will install the required Python packages in a virtual environment.
 
+## Disk Support for Materialize on Azure
+
+This module supports configuring disk support for Materialize on Azure using **local NVMe SSDs** available in specific VM families, along with **OpenEBS** and `lgalloc` for volume management.
+
+### Recommended Azure VM Types with Local NVMe Disks
+
+Materialize benefits from fast ephemeral storage and recommends a **minimum 2:1 disk-to-RAM ratio**. The [Lsv3-series](https://learn.microsoft.com/en-us/azure/virtual-machines/lsv3-series) virtual machines offer high-throughput local NVMe SSD storage and are ideal for performance-intensive workloads.
```

Comment:
+1 to Justin's comment about instance types.

The E* series' also support local drives, but offer much more ram.